package.json

```diff
@@ -26,17 +26,19 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "ember-auto-import": "^1.6.0",
     "ember-cli-babel": "^7.22.1",
```

Comment:
Relevant to my comments above: worth double-checking, but I'm nearly positive that this is recent enough that private class fields support is available by default!